src/main/java/org/jabref/logic/openoffice/action/EditMerge.java

```diff
@@ -93,21 +93,11 @@ public static boolean mergeCitationGroups(XTextDocument doc, OOFrontend frontend
         return madeModifications;
     }
 
-    private static class JoinableGroupData {
-        /**
-         * A list of consecutive citation groups only separated by spaces.
-         */
-        List<CitationGroup> group;
-
-        /**
-         * A cursor covering the XTextRange of each entry in group (and the spaces between them)
-         */
-        XTextCursor groupCursor;
-
-        JoinableGroupData(List<CitationGroup> group, XTextCursor groupCursor) {
-            this.group = group;
-            this.groupCursor = groupCursor;
-        }
+    /**
+     * @param group : A list of consecutive citation groups only separated by spaces.
```

Comment:
Please follow the correct syntax

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format

Discussion at https://stackoverflow.com/a/16138292/873282

If you want, you can search whether checkstyle supports JavaDoc checks or if there are other ways to check for wrong syntax.

Note that IntelliJ should align the text - which is not the case in your thing. If you mark the two lines and press  <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>L</kbd>, what happens?